app/assets/stylesheets/nav.scss

```diff
@@ -10,10 +16,31 @@ $button-color: #1E90FF;
   list-style-type: none;
 }
 
+.top-bar {
+  background-color: $bar;
+}
+
+h3{
+  font-family: $font-sans;
+}
+
+h2{
+  color: $white;
+}
+
+body{
+  background-color: $background;
+  font-family: $font-sans;
+}
+
+body.home{
```

Comment:
Merge rule `body.home` with rule on line 31<br>Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space